student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
```

Comment:
- [ ] @hetrox8 Great work adding the student class, when referencing a file it's ruby best practise to use require_relative instead of require, kindly consider using require_relative to reference person class.